mw-webapp/src/utils/useAuth.ts

```diff
@@ -20,5 +38,6 @@ export const useAuth = () => {
       alert(errorMessage);
     }
   };
-  return handleGoogleSignIn;
+
+  return {handleGoogleSignIn, handleLogout, writeNewUserData, auth};
 };
```

Comment:
Also maybe better moveout these functions to keep single responsibility

Reply:
After that, the existence of the useAuth hook became senseless and I deleted it.
Done